identity-service/src/app.js

```diff
@@ -43,7 +43,17 @@ class App {
     await new Promise(resolve => setTimeout(resolve, 2000))
     await this.runMigrations()
     await this.getAudiusAnnouncements()
-    await this.configureAudiusInstance()
+
+    // exclude these init's if running tests
+    if (!config.get('isTestRun')) {
+      const audiusInstance = await this.configureAudiusInstance()
+      await this.notificationProcessor.init(
```

Comment:
@hareeshnagaraj i moved the notification processor here so we just need one if check. is this okay or should i have it after server port init?

Reply:
i think this should be fine, everything works on local?

Reply:
yeah seems to work fine. i can do another pass of tests before merging